package.json

```diff
@@ -8,7 +8,6 @@
   },
   "scripts": {
     "preinstall": "npm install pm2 -g",
-    "heroku-postbuild": "npm run build",
```

Comment:
It is independent from `pm2`, right?

Reply:
Yes, this script is for Heroku platform, so it's useless for our deployment platform.

Reply:
Decouple this commit from pull request. Let's focus on pm2-specific changes.